src/components/AddBook.jsx

```diff
@@ -0,0 +1,9 @@
+const AddBook = () => (
+  <div>
+    <input type="text" placeholder="title" />
+    <input type="text" placeholder="author" />
+    <button type="submit">Add Book</button>
+  </div>
+);
```

Comment:
- [ ] Great job for creating the form but I can see that you didn't render it in books component since it should be after the list a form with inputs so kindly import it to be after the list as mentioned in the requirements 🙏🏻✨

![image](https://github.com/Nasirkhan294/bookstore/assets/58553711/5c95137e-1259-4917-9e60-0d6733346b31)


- The list of the books then the form , style is not required but just the wireframe ✅🔥✨⭐
![image](https://github.com/Nasirkhan294/bookstore/assets/58553711/7760b207-7fdf-47b6-92e1-7d45144e5145)